.github/ISSUE_TEMPLATE/4_KIT_release_checks.md

```diff
@@ -3,25 +3,31 @@ name: KIT Release Checks
 about: Use this template to track KIT release checks
 title: "[KIT NAME] Release Checks"
 labels: kit
+milestone: "24.05"
```

Comment:
Hi @kelaja,

could you please remove this item this will make the template not usable, because `milestone` is not a default value for issue templates

Reply:
@FaGru3n Thanks for reminding me. I applied your proposal, kindly approve.